index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="app.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    
+    <header class="navlogo">
+        <img src="images/My Logo.png" alt="Mylogo">
+
+        <u class="nav-icon">
+            <li class="Hamburger">
+                <img src="images/Union.png" alt="menuicon">
+                <a href="#">
+                    <i class="fa-thin fa-bars"></i>
+                </a>
+            </li>
+        </u>
+    </header>
+    
+    <!--Header Section-->
+    <section class="header_info_section">
+
+        <h1 class="greeting-header">Hey There. <br> I'm Mannie</h1>
+        <h2 class="SupportingText">I'am a Software Developer</h2>
+
+            <p class="Description"> I can help you build a product, 
+            feature or website look through some of my work and 
+            experience! if you like what you see and have a project 
+            you need coded, don't hesitate to contact me.</p>
+
+            <ul class="social-icons">
+                <li class="Github">
+                    <img src="images/Git.png" alt="GitHub icon">
+        
+                </li>
+                <li class="linkedin">
+                    <img src="images/Linkedin.png" alt="Linkedin icon">
+        
+                </li>
+                <li class="Angel List">
+                    <img src="images/AngelList.png" alt="Angel icon">
+                    <a href="#">
+                        <i class="fa fa-AngelList" area-hidden="true"></i>
+                    </a>
+                </li>
+                <li class="Twitter">
+                    <img src="images/Twitter.png" alt="Twitter icon">
+                    <a href="#">
+                        <i class="fa fa-Twitter" area-hidden="true"></i>
+                    </a>
+                </li>
+                <li class="Medium">
+                    <img src="images/Medium.png" alt="Medium">
+                    
+                </li>
```

Comment:
- [ ] As requested by the previous reviewer, Kindly make sure to wrap all your image icons into the Anchor `a` tag. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using the `href` attribute.